lib/util/miq-password.rb

```diff
@@ -156,6 +144,13 @@ def self.ez_load(filename)
     File.exist?(filename) && EzCrypto::Key.load(filename)
   end
 
+  def self.ez_load_v0(filename)
```

Comment:
Think all of these protected methods are accessible by `MiqAePassword` but not really used there.

You thinking we should just move all of these over to private?